src/js/api/set-config.js

```diff
@@ -58,6 +70,17 @@ export default (controller, newConfig) => {
             case 'stretching':
                 model.set(field, newValue);
                 break;
+            case 'floating':
+                switch (newValue.mode) {
+                    case 'always':
+                    case 'never':
+                    case 'notVisible':
+                        controller._view.initFloatingBehavior();
```

Comment:
Why is the model set above, but here we directly call out to the view? Couldn't we just `model.set('floating'` here and respond to change in the view or floating controller?  I'm worried there could be a disconnect between things that respond to model change, and this code running afterwards.